test/inductor/test_flex_attention.py

```diff
@@ -912,14 +912,15 @@ def composed_score_mod(score, b, h, m, n):
         self.run_test(composed_score_mod, dtype)
 
     @supported_platform
+    @skip("Not support compiled flex attention with training bias")
```

Comment:
Verified eager flex attention works well, it errors out at compiled version.

Reply:
maybe x fail, with good message?